apps/api/src/routes/application-router.ts

```diff
@@ -233,4 +236,73 @@ applicationRouter.post('/applications/approve', jsonParser, async (req, res) =>
 	}
 });
 
+applicationRouter.post('/applications/reject', jsonParser, async (req, res) => {
+	const { applicationId } = req.body;
+
+	if (!applicationId) {
+		res.status(400).json({ message: 'Application ID is required.' });
+	}
+
```

Comment:
Both of these `if` blocks send status `400` and concern `applicationId` so I think it would make sense to combine them:

```
	if (!applicationId || isNaN(parseInt(applicationId))) {
		res.status(400).json({ message: 'Invalid request. ApplicationId is required and must be a valid number.',
			errors: 'MissingOrInvalidParameters', });
	}
```